librabbitmq/win32/threads.h

```diff
@@ -34,7 +34,12 @@
 #ifndef WIN32_LEAN_AND_MEAN
 # define WIN32_LEAN_AND_MEAN
 #endif
+
+#if (defined(__MINGW32__) || defined(__MINGW64__))
```

Comment:
OK. Sure, will do the changes. Need some time to validate the build with Mingw-w64 cross compiler though.